### PR TITLE
During bootstrap include all headers

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

File: src/Rootfs.jl [hunk 1]
```diff
@@ -549,7 +549,14 @@ function choose_shards(p::Platform;
             push!(shards, CompilerShard("Rootfs", find_latest_version("Rootfs"), host_platform, archive_type))
         end
         if :platform_support in bootstrap_list
-            push!(shards, CompilerShard("PlatformSupport", find_latest_version("PlatformSupport"), host_platform, archive_type; target=p))
+            for target in supported_platforms()
+                push!(shards, CompilerShard("PlatformSupport", find_latest_version("PlatformSupport"), host_platform, archive_type; target=target))
+            end
+        end
+        if :glibc in bootstrap_list
```
Comment (vchuravy) [inline src/Rootfs.jl:556]: @staticfloat Maybe it makes sense to separate the glibc headers from the GCC shard?

Comment (vchuravy) [reply src/Rootfs.jl:556]: To be clear, without this I got:

```
sandbox:/opt/aarch64-linux-gnu/aarch64-linux-gnu/sys-root # find /opt/  -name "stdlib.h"
/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/include/c++/4.2.1/tr1/stdlib.h
/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/include/c++/v1/stdlib.h
/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/include/stdlib.h
/opt/x86_64-unknown-freebsd11.1/x86_64-unknown-freebsd11.1/sys-root/include/c++/v1/stdlib.h
/opt/x86_64-unknown-freebsd11.1/x86_64-unknown-freebsd11.1/sys-root/include/c++/v1/tr1/stdlib.h
/opt/x86_64-unknown-freebsd11.1/x86_64-unknown-freebsd11.1/sys-root/include/stdlib.h
/opt/i686-w64-mingw32/i686-w64-mingw32/sys-root/include/stdlib.h
/opt/x86_64-w64-mingw32/x86_64-w64-mingw32/sys-root/include/stdlib.h
```

and `compiler-rt` uses `stdlib.h`.

Comment (staticfloat) [reply src/Rootfs.jl:556]: We don't want to add GCC shards to bootstrapping; that kind of defeats the purpose of bootstrapping.

We could add the glibc headers to PlatformSupport.

**Comment by vchuravy** [comment]
Forgotten why I wanted this.

**Pull request closed without merging**